misc/extract_knowledge_from_msi_table.py

```diff
@@ -44,10 +44,18 @@ def create_dtc_dictionary(path: str) -> dict:
 
     data = pandas.read_excel(path, sheet_name="DTC - Element ID - Baum")
 
+    def get_existing_fault_conditions() -> list:
```

Comment:
Is this supposed to be a nested function?

Reply:
Yes